functests/storageclusterinitialization_test.go

```diff
@@ -143,53 +143,53 @@ var _ = Describe("StorageClusterInitialization", func() {
 		}, 15*time.Second, 1*time.Second).ShouldNot(gomega.HaveOccurred())
 	}
 
-	cephObjectStoreUserModify := func(shouldDelete bool) {
-		objStoreUser := &cephv1.CephObjectStoreUser{}
-		key := crclient.ObjectKey{Namespace: namespace, Name: fmt.Sprintf("%s-cephobjectstoreuser", name)}
-		err := client.Get(context.TODO(),
-			key,
-			objStoreUser,
-		)
-		Expect(err).To(BeNil())
-
-		// modifying the user store name
-		if shouldDelete {
-
-			err = client.Delete(context.TODO(), objStoreUser)
-			Expect(err).To(BeNil())
-		} else {
-			objStoreUser.Spec.Store = "fake"
-			err = client.Update(context.TODO(), objStoreUser)
-			Expect(err).To(BeNil())
-		}
-		uidMap[objStoreUser.UID] = ""
-	}
-
-	cephObjectStoreUserExpectReconcile := func(expectDelete bool) {
-		objStoreUser := &cephv1.CephObjectStoreUser{}
-		key := crclient.ObjectKey{Namespace: namespace, Name: fmt.Sprintf("%s-cephobjectstoreuser", name)}
-
-		gomega.Eventually(func() error {
-			err := client.Get(context.TODO(),
-				key,
-				objStoreUser,
-			)
-			if err != nil {
-				return err
-			}
-
-			if objStoreUser.Spec.Store == "fake" {
-				return fmt.Errorf("Waiting on ceph object store user to reconcile changes")
-			}
-
-			if !expectDelete {
-				_, ok := uidMap[objStoreUser.UID]
-				Expect(ok).To(BeTrue())
-			}
-
-			return nil
-		}, 15*time.Second, 1*time.Second).ShouldNot(gomega.HaveOccurred())
-	}
+	// cephObjectStoreUserModify := func(shouldDelete bool) {
+	// 	objStoreUser := &cephv1.CephObjectStoreUser{}
+	// 	key := crclient.ObjectKey{Namespace: namespace, Name: fmt.Sprintf("%s-cephobjectstoreuser", name)}
+	// 	err := client.Get(context.TODO(),
```

Comment:
Not sure that commenting out massive blocks of code is a clean way to disable the tests... I think we want a better way.

Reply:
Agreed. Changed to include platform detection in the functests